README.md

```diff
@@ -1,8 +1,72 @@
-<img src="https://raw.githubusercontent.com/DiamondLightSource/python-copier-template/main/docs/images/dls-logo.svg"
-     style="background: none" width="120px" height="120px" align="right">
+# Diamond Image Service
 
-[![CI](https://github.com/DiamondLightSource/ptycho-reduction/actions/workflows/ci.yml/badge.svg)](https://github.com/DiamondLightSource/ptycho-reduction/actions/workflows/ci.yml)
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+## Introduction
```

Comment:
At this bit I would suggest adding links to the docs in the docs folder.

Reply:
I have moved the links here